prov/efa/src/efa_msg.c

```diff
@@ -286,7 +286,6 @@ static void efa_post_send_sgl(struct efa_ep *ep, const struct fi_msg *msg,
 	struct efa_mr *efa_mr;
 	struct ibv_send_wr *wr = &ewr->wr;
 	struct ibv_sge *sge;
-	size_t sgl_idx = 0;
```

Comment:
This change is good. But probably should go into a separate commit

Reply:
ACK